comms/shared/utils/discovery.go

```diff
@@ -0,0 +1,53 @@
+package utils
+
+import (
+	"crypto/ecdsa"
+	"encoding/json"
+
+	"comms.audius.co/shared/peering"
+	"github.com/ethereum/go-ethereum/crypto"
+	"golang.org/x/exp/slices"
+)
+
+func IsValidDiscoveryNode(wallet string) bool {
+
+	p := peering.New(nil)
+
+	nodes, err := p.GetDiscoveryNodes()
+	if err != nil {
+		return false
+	}
+
+	wallets := make([]string, len(nodes))
+	for _, value := range nodes {
+		wallets = append(wallets, value.DelegateOwnerWallet)
+	}
+
+	includes := slices.Contains(wallets, wallet)
```

Comment:
`string.ToLower()` on `wallet`? Do you have an example

Reply:
Yeah on wallet like slices.Includes(wallets, strings.ToLower(wallet)). Also when appending to the list just in case it would be good to do wallets = append(wallets, strings.ToLower(value.DelegateOwnerWallet))

Reply:
why is that? it sounds sus